src/V8JsiRuntime.cpp

```diff
@@ -69,27 +76,31 @@ class TaskRunnerAdapter : public v8::TaskRunner {
       : taskRunner_(std::move(taskRunner)) {}
 
   void PostTask(std::unique_ptr<v8::Task> task) override {
-    taskRunner_->postTask(std::make_unique<TaskAdapter>(std::move(task)));
+    taskRunner_->postTask(node::static_unique_pointer_cast<JSITask>(

```

Comment:
>node::static_unique_pointer_cast<JSITask> [](start = 26, length = 41)

Is this needed? I think you noticed this code is not actually used.

Reply:
The VS Code IntelliSense was unhappy about it and I saw the opportunity to use the Node.js function to fix it.